src/app/contact/selectService.js

```diff
@@ -48,11 +48,14 @@ const post = async (req, res) => {
     model.csrfToken = req.csrfToken();
     return res.render('contact/views/selectService', model);
   }
-
+const serviceIdentity = model.selectedService.split(',');
   if (model.selectedService === 'other') {
     return res.redirect('/contact/submit?type=service-access&service=Other');
-  } else {
-    return res.redirect(`service/${model.selectedService}`);
+  }else if (serviceIdentity[1] === 'Course Directory') {
```

Comment:
needs to be defensively coded, if only one selected service this will fail with `ReferenceError`